deploy/stacks/run_if.py

```diff
@@ -17,16 +19,20 @@ def no_decorated(f):
     return fn, staticmethod if static_func else no_decorated
 
 
-def run_if(active_property: str):
+def run_if(active_properties: List[str]):
```

Comment:
Also in the original one, I got a question shouldn't `active_properties` be of type List[bool]?

Reply:
It is possible to achieve. But it would duplicate the code a little in all files since we would need to get the value of properties (`deploy_config.get_property`) before calling the decorator. I don't see that we would benefit much by passing List[bool] at this point of time 😄  (Of course, it can easily change later with a new reqs)